### ISSUE TITLE
label erro with dev branch

### ISSUE THREAD

**Issue by bobleegogogo** [GIScience/ohsome2label#7]
the command of "ohsome2label label" with default config gives me the following error:

![bug](https://user-images.githubusercontent.com/33456076/144273785-ad0e11eb-265d-488a-b3e3-10cb753deee3.JPG)

I think it is related to 

https://github.com/GIScience/ohsome2label/blob/19f456ceabf2a5ffe2e83b29eed5fc707ffd8b27/ohsome2label/label.py#L354

@wuzyzy could you maybe check the dev branch?

**Comment by wuzyzy** [comment]
Yes, I change the pipeline. You should use `ohsome2label tile` before using `ohsome2label label` and `ohsome2label download`. Here is  a function called `get_tile_list`  in [label.py](https://github.com/GIScience/ohsome2label/blob/dev/ohsome2label/label.py), I divided it from `gen_label` . Cause I found that when I use the user-defined WMTS serveice for downloading satellite image, the old version cannot run to the end when the area of vector is exceed the region covered by the WMTS without a tile_list. 
Sorry I forgot to mention it in git commit. I think we should change the readme so users can use it correctly.

**Comment by bobleegogogo** [comment]
Current status: 

- new function "viewer" in dev branch
- new command link logic in dev branch --> ohsome2label vector, tile, image, label

Aim to solve:

- bad annotation without valid image from the used WMTS
- index error when having multiple classes objects/classes 

Decision:

- continue with seperated branch in dev
- release main branch as  [https://pypi.org/project/ohsome2label/1.1.3/](url)
